WooCommerce/Classes/ViewRelated/Dashboard/MyStore/Cells/ProductTableViewCell.swift

```diff
@@ -61,10 +71,29 @@ extension ProductTableViewCell {
         )
         priceText = statsItem?.formattedTotalString
 
+        /// Set `center` contentMode to not distort the placeholder aspect ratio.
+        /// After a sucessfull image download set the contentMode to `scaleAspectFill`
+        productImage.contentMode = .center
```

Comment:
Do you think it's possible to change the size of `.productPlaceholderImage` instead so we don't need to do the `productImage.contentMode` dance?

Reply:
I was thinking on how to accomplish your suggestion and I think the way to resize the image to mimic the product tab would be to manually draw the icon at the center location of a bigger drawing context, but that seems a bit over complicated for such a simple task. I would rather do the `contentMode` dance, what do you think?

Reply:
Yeah, I was thinking about [adding insets](https://stackoverflow.com/questions/20021478/add-transparent-space-around-a-uiimage/31240900). It's additional code but it simplifies this part which I believe is worth it. 

Anyway, it's fine. Let's keep this. 🙂

Reply:
👍